### DISCUSSION TITLE
Wlan AP Passwort

### DISCUSSION THREAD

**Discussion by AnHardt** [open3e/open3e#75] [Q&A] [answered]
Hallo,.
Habe eine Vitovalor PT2. Bevor ich in einen CAN-Adapter investiere würde ich gern den Weg ausprobieren, welche Datenpunkte ich überhaupt auslesen kann. Deswegen würde ich gern einen Vortest über den AP der Heizung probieren.
Leider scheitere ich schon beim Verbinden mit dem AP. Das WLAN zu finden ist einfach. Aber wie ist das Passwort?

**Comment by philippoo66** [comment, marked as answer]
https://github.com/open3e/open3e/wiki/038-open3e-%C3%BCber-WLAN-Access-Point-nutzen

"Den WPA-Schlüssel liest man vom Aufkleber ab."

Grüsse!

**Reply by AnHardt** [reply]
Danke, diese Hürde wäre genommen.
